test/fuzzTesting/CMakeLists.txt

```diff
@@ -1,19 +1,5 @@
 cmake_policy(SET CMP0015 NEW)
 
-file(GLOB HEADERS "*.h")
-include_directories(BEFORE ${JSONCPP_INCLUDE_DIRS})
-include_directories(BEFORE ..)
-include_directories(BEFORE ../..)
-include_directories(${Boost_INCLUDE_DIRS})
-include_directories(${CRYPTOPP_INCLUDE_DIRS})
-include_directories(${JSON_RPC_CPP_INCLUDE_DIRS})
+aux_source_directory(. SRCS)
```

Comment:
I know. I've seen @debris is moving away from aux_source_directory in cmake cleanups. cmake advices not to use it for this cases.

It can stay for now, but take that into consideration next time.